### PR TITLE
Add draft missing data outline

### PR DESCRIPTION
**Pull request by awunderground**

### CODE REVIEW

File: outline.Rmd [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+$$p(\theta|x, y_{obs}) = p(\theta|x, y_{obs}, I)$$
+
+\vspace{0.25in}
+
+\textbf{Definition:} \textit{missing not at random (MNAR)} -- If the missing data mechanism depends on the missing values. Neither MCAR or MAR holds. The data are missing for reasons that are unknown to us. 
+
+## Creating Missing Data
+
+We assume that there is no unit missingness. Our observations are either a simple random sample from the population or the complete population. Our approach will only focus on unit missingness, where values within an observation are missing. We will start with our complete data and then construct different obscured data sets with different types of mssingness.
```
Comment (lbotti319) [inline outline.Rmd:66]: What is "unit missingness"

**Review by lbotti319 — COMMENTED** [review]
I'm not seeing the difference between MAR with non-ignorable mechanism and MNAR.

In the creating missing data section for MAR with non-ignorable mechanism you use the example of high weight teens. In the definition MNAR, the missing data depends on the missing values. Higher weight teens would be more likely to have missing values, right?

File: outline.Rmd (hunk 1 above)
Comment (awunderground) [reply outline.Rmd:66]: Unit missingness is when an entire observation (row) is missing. Imagine a person just not responding to an entire survey.

Item missingness is when a specific value (cell) is missing. Imagine a person responding to a survey but skipping a question.

Comment (lbotti319) [reply outline.Rmd:66]: In that case
> Our approach will only focus on unit missingness
should be
> Our approach will only focus on item missingness
right?

**Review by inespancorbo — APPROVED** [review]
I'm ok with this too. We can just have it in main so we can all start adding

File: outline.Rmd (hunk 1 above)
Comment (awunderground) [reply outline.Rmd]: Fixed!

**Review by lbotti319 — APPROVED** [review]

**Comment by inespancorbo** [comment]
Also I just realized that the definitions are in term of Y missing and observed. Are we going with missing in the covariates or in the response?

**Comment by awunderground** [comment]
I think y just means all of the data. y is the same size as the inclusion matrix in the multivariate case.

**Pull request merged**